pkg/auth/dependency/webapp/service.go

```diff
@@ -378,6 +378,8 @@ func (s *Service) deriveRedirectPath(graph *newinteraction.Graph, edges []newint
 		return "/oob_otp"
 	case *nodes.NodeCreateAuthenticatorOOBSetup:
 		return "/oob_otp"
+	case *nodes.NodeVerifyIdentity:
+		return "/verify_user"
```

Comment:
What we are verifying is actual login ID identity. Maybe rename it to `/verify_identity`?